x-pack/functionbeat/provider/aws/aws/kinesis.go

```diff
@@ -70,7 +70,7 @@ func (s *startingPosition) Unpack(str string) error {
 func (s *startingPosition) String() string {
 	v, ok := mapStartingPositionReverse[*s]
 	if !ok {
-		panic("unknown starting position: " + string(*s))
+		panic("unknown starting position: " + fmt.Sprint(*s))
```

Comment:
This fix seems somewhat unrelated to the ECS updates (which btw there are two [other](https://github.com/andrewkroh/beats/blob/a5b11215dfc214890c28a9e4dddb6fa200ea55f8/.nogo.json#L171-L175) instances of this problem).